core/src/main/scala/org/broadinstitute/dsde/rawls/entities/compact/CompactEntitySerialization.scala

```diff
@@ -49,7 +49,7 @@ trait CompactEntitySerialization {
     }
 
   val slickAttrsPath: String = s"$$.${ATTRS_KEY}"
-  def slickAttributePath(attributeName: String): String = s"${slickAttrsPath}.${attributeName}"
+  def slickAttributePath(attributeName: String): String = s"""${slickAttrsPath}."${attributeName}""""
```

Comment:
the extra double-quotes around the attribute name are necessary for attributes that contain a colon, e.g. `import:timestamp`